functions/Save-DbaCommunitySoftware.ps1

```diff
@@ -15,6 +15,7 @@ function Save-DbaCommunitySoftware {
         * DarlingData: https://github.com/erikdarlingdata/DarlingData
         * SQLWATCH: https://github.com/marcingminski/sqlwatch/releases
         * WhoIsActive: https://github.com/amachanic/sp_whoisactive/releases
+        * DbaMultitool: https://github.com/LowlyDBA/dba-multitool/releases
```

Comment:
Maybe change the "t" in "tool" to upper case like in the validate set of the software parameter?